### ISSUE TITLE
Cannot train Relation Extraction with LayoutXLM on custom dataset

### ISSUE THREAD

**Issue by NMT201** [PaddlePaddle/PaddleOCR#9969]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubuntu 22.04, CUDA 11.7
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：paddlepaddle-gpu==2.4.2.post117, paddlenlp==2.5.2

- config file:
 Global:
  use_gpu: False
  epoch_num: &epoch_num 130
  log_smooth_window: 10
  print_batch_step: 10
  save_model_dir: ./output/re_business_registration
  save_epoch_step: 2000

  eval_batch_step: [ 0, 19 ]
  cal_metric_during_train: False
  save_inference_dir:
  use_visualdl: False
  seed: 2022
  infer_img: ppstructure/docs/kie/input/zh_val_21.jpg
  save_res_path: ./output/re/xfund_zh/with_gt
  kie_rec_model_dir: 
  kie_det_model_dir:

Architecture:
  model_type: kie
  algorithm: &algorithm "LayoutXLM"
  Transform:
  Backbone:
    name: LayoutXLMForRe
    pretrained: True
    use_visual_backbone: true
    mode: vi
    checkpoints:

Loss:
  name: LossFromOutput
  key: loss
  reduction: mean

Optimizer:
  name: AdamW
  beta1: 0.9
  beta2: 0.999
  clip_norm: 10
  lr:
    learning_rate: 0.00005
    warmup_epoch: 10
  regularizer:
    name: L2
    factor: 0.00000
    
PostProcess:
  name: VQAReTokenLayoutLMPostProcess

Metric:
  name: VQAReTokenMetric
  main_indicator: hmean

Train:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/train
    label_file_list: 
      - train_data//train.txt
    ratio_list: [ 1.0 ]
    transforms:
      - DecodeImage: # load image
          img_mode: RGB
          channel_first: False
      - VQATokenLabelEncode: # Class handling label
          contains_re: True
          algorithm: *algorithm
          class_path: &class_path train_data/class_list.txt
          use_textline_bbox_info: &use_textline_bbox_info True
          order_method: &order_method "tb-yx"
      - VQATokenPad:
          max_seq_len: &max_seq_len 512
          return_attention_mask: True
      - VQAReTokenRelation:
      - VQAReTokenChunk:
          max_seq_len: *max_seq_len
      - TensorizeEntitiesRelations:
      - Resize:
          size: [224,224]
      - NormalizeImage:
          scale: 1
          mean: [ 123.675, 116.28, 103.53 ]
          std: [ 58.395, 57.12, 57.375 ]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: [ 'input_ids', 'bbox','attention_mask', 'token_type_ids', 'entities', 'relations'] # dataloader will return list in this order
  loader:
    shuffle: True
    drop_last: False
    batch_size_per_card: 1
    use_shared_memory: False
    num_workers: 4

Eval:
  dataset:
    name: SimpleDataSet
    data_dir: train_data/val
    # data_dir: train_data/test
    label_file_list:
      - train_data/val.txt
    transforms:
      - DecodeImage: # load image
          img_mode: RGB
          channel_first: False
      - VQATokenLabelEncode: # Class handling label
          contains_re: True
          algorithm: *algorithm
          class_path: *class_path
          use_textline_bbox_info: *use_textline_bbox_info
          order_method: *order_method
      - VQATokenPad:
          max_seq_len: *max_seq_len
          return_attention_mask: True
      - VQAReTokenRelation:
      - VQAReTokenChunk:
          max_seq_len: *max_seq_len
      - TensorizeEntitiesRelations:
      - Resize:
          size: [224,224]
      - NormalizeImage:
          scale: 1
          mean: [ 123.675, 116.28, 103.53 ]
          std: [ 58.395, 57.12, 57.375 ]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: [ 'input_ids', 'bbox', 'attention_mask', 'token_type_ids', 'entities', 'relations'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1
    num_workers: 4
    use_shared_memory: False

- 完整报错/Complete Error Message：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/79817902/92031266-bf7c-43e1-901e-a2af921f5468)

**Comment by happybuby** [comment]
是不是标注数据集的问题啊

**Comment by NMT201** [comment]
@happybuby I've checked my dataset several and not see any problem

**Comment by NMT201** [comment]
@happybuby Here my train annotation file, can you help me find out [this](https://drive.google.com/file/d/1bxVMGMoN5jXKUXRHVSWxvlvNTYVJxs4M/view?usp=sharing)

**Comment by phamkhactu** [comment]
@NMT201 Can you give me your datasets, I have faced with another [problem](https://github.com/PaddlePaddle/PaddleOCR/issues/8643).
I can try and find out which problems come.

**Comment by happybuby** [comment]
@NMT201 you can give me the file,i will try

**Comment by NMT201** [comment]
> @happybuby here

**Comment by NMT201** [comment]
@happybuby [Here](https://drive.google.com/file/d/1bxVMGMoN5jXKUXRHVSWxvlvNTYVJxs4M/view)

**Comment by happybuby** [comment]
when you replace the labels with your custom labels,there are some problems because of hardcode

**Comment by NMT201** [comment]
what you mentioned about

**Comment by happybuby** [comment]
in the program,the label [problem] and [answer] is hardcode,you need change them as you need

**Comment by happybuby** [comment]
in the program,the label [question] and [answer] is hardcode,you need change them as you need

**Comment by NMT201** [comment]
Can you show me where to change it

**Comment by happybuby** [comment]
you can search in vqa_token_relation.py

**Comment by NMT201** [comment]
It work. Thanks

**Comment by happybuby** [comment]
you will encounters more other problems while appling paddle-re in projects ,I am trying

**Comment by NMT201** [comment]
> 

:D

**Comment by NMT201** [comment]
@happybuby You're right :D I got that problem again. Do it require every label must appear in every single image annotation ?

**Comment by ariefwijaya** [comment]
@NMT201 Hi, how did you solve the problem?

**Comment by NMT201** [comment]
> @NMT201 Hi, how did you solve the problem?
could you give me your annotation file

**Comment by L-Q-K** [comment]
> It work. Thanks

Facing the same issue, can you show me how to fix it
